packages/paypal-commerce-integration/src/paypal-commerce/paypal-commerce-customer-strategy.spec.ts

```diff
@@ -213,7 +217,7 @@ describe('PayPalCommerceCustomerStrategy', () => {
 
             expect(paypalSdk.Buttons).toHaveBeenCalledWith({
                 fundingSource: paypalSdk.FUNDING.PAYPAL,
-                style: { height: 40 },
+                style: { height: 40, color: StyleButtonColor.silver, label: 'checkout' },
```

Comment:
Hi @bc-dronov, I was refering here. So for a `customer-strategy`, which is placed at the top of the checkout page, the height of a button is designed as `36px`. 

Just this one place.

Reply:
Done!

Reply:
Fantastic, thank you.